answer_rocket/graphql/operations/chat.gql

```diff
@@ -1,103 +1,100 @@
-
 mutation AskChatQuestion(
-    $copilotId: UUID!
-    $question: String!
-    $threadId: UUID
-    $skipReportCache: Boolean
-    $dryRunType: ChatDryRunType
+  $copilotId: UUID!
+  $question: String!
+  $threadId: UUID
+  $skipReportCache: Boolean
+  $dryRunType: ChatDryRunType
+  $modelOverrides: [ModelOverride]
 ) {
-    askChatQuestion(
-        copilotId: $copilotId,
-        question: $question,
-        threadId: $threadId
-        skipReportCache: $skipReportCache
-        dryRunType: $dryRunType
-    ) {
-        id
-        threadId
-        answer {
-            ...ChatResultFragment
-        }
+  askChatQuestion(
+    copilotId: $copilotId
+    question: $question
+    threadId: $threadId
+    skipReportCache: $skipReportCache
+    dryRunType: $dryRunType
+    modelOverrides: $modelOverrides
+  ) {
+    id
+    threadId
+    answer {
+      ...ChatResultFragment
     }
+  }
 }
 
 mutation QueueChatQuestion(
-    $threadId: UUID!
-    $question: String!
-    $skipCache: Boolean
+  $threadId: UUID!
+  $question: String!
+  $skipCache: Boolean
```

Comment:
does this mutation not also need to support these overrides? This mutation also asks a question, but it immediately returns an id to use to poll for the result instead.